test/data/events.json

```diff
@@ -62,32 +62,6 @@
     "time": "2023-02-14T19:21:14.113113Z",
     "type": "com.wasmcloud.lattice.health_check_status"
   },
```

Comment:
Can you please add a sample ActorScaled and ActorScaleFailed event in this file? Just want to make sure we catch any deserialization problems as we make changes